packages/knip/fixtures/plugins/react-router/app/routes.ts

```diff
@@ -0,0 +1,7 @@
+export default [
+  { file: "routes/home.tsx", index: true },
+  {
+    file: "routes/layout.tsx",
+    children: [{ file: "routes/another-route.tsx" }],
+  },
+];
```

Comment:
seeing things in the docs like `route("/", "./home.tsx")` - do we cater for that as well? or how's that returned when loading `app/routes.ts`?

Reply:
`route("/", "./home.tsx")` returns a simple object containing `{ file: "./home.tsx", path: "/" }`. Since we are joining the path like this: `join(appDir, route.file)` and the `routes.ts` file is defined in `appDir` this is catered for.